cmd/process-agent/collector_test.go

```diff
@@ -98,3 +100,45 @@ func TestHasContainers(t *testing.T) {
 	assert.Equal(2, getContainerCount(&collectorRealTime))
 	assert.Equal(1, getContainerCount(&collectorContainerRealTime))
 }
+
+func TestDisableRealTime(t *testing.T) {
+
+	tests := []struct {
+		name            string
+		disableRealtime bool
+		expectedChecks  []checks.Check
+	}{
+		{
+			name:            "true",
+			disableRealtime: true,
+			expectedChecks:  []checks.Check{checks.Process, checks.Container},
+		},
+		{
+			name:            "false",
+			disableRealtime: false,
+			expectedChecks:  []checks.Check{checks.Process, checks.Container, checks.RTContainer},
```

Comment:
Shouldn't we expect `RTProcessCheckName` here as well?

Reply:
There's no `RTProcess` anymore, it's the same `Process` check handling both.

Reply:
Thanks for clarifying it!